src/templates/actors/character-sheet2e.hbs

```diff
@@ -31,7 +31,7 @@
             {{localize 'sta.actor.character.assignment'}}
           </div>
         </div>
-        <div class="row">
+        <div class="row name-species-row">
```

Comment:
I'm unclear on the purpose of "name-species-row" as a classname, I didn't find related CSS.

Reply:
It was required to hook into for the LCARS UI module to keep working with the changes, a side benefit of being involved in both projects now.